enable_location.sh

```diff
@@ -112,3 +113,49 @@ location ${LOCATION} {
 
 }
 EOF_LOCATION_CONF
+
+# If Static Caching is enabled, add Cache Config for the Server Block...
+if [ -n "${PROXY_STATIC_CACHING:-}" ]; then
+ESCAPED_LOCATION=$(eval "echo $LOCATION | sed 's;/;\\\/;g'")
+
+cat << EOF_SERVERCACHE_CONF >> /etc/nginx/conf/locations/${LOCATION_ID}.conf
+# Allow Nginx to cache static assets - follow the same proxy config as above.
+location ~* ^${ESCAPED_LOCATION}(.+)\.(jpg|jpeg|gif|png|svg|ico|css|bmp|js|html|htm|ttf|otf|eot|woff|woff2)$ {
```

Comment:
Does anyone see anything funny about this regex pattern that might be causing Nginx not to apply the caching rules to  CSS and JS assets? I see a "304 Not Modified" response in my Browser's inspector, so could that be hiding the Cache headers from the response?

I _think_ this is fine, but just wanted to get a second opinion.

Reply:
Have you tried this out various values of `${ESCAPED_LOCATION}` in something like https://regex101.com?

Reply:
> Have you tried this out various values of `${ESCAPED_LOCATION}` in something like https://regex101.com?

<img width="824" alt="Screenshot 2020-12-02 at 12 48 07" src="https://user-images.githubusercontent.com/68009085/100874607-bddf4280-349c-11eb-99cf-b0a5aee5d064.png">

That's what I get.